src/modules/app/reducers/footer-height.js

```diff
@@ -1,11 +1,14 @@
 import { UPDATE_FOOTER_HEIGHT } from 'modules/app/actions/update-footer-height'
```

Comment:
We actually don’t use this state value any longer in the new layout.

You’re safe to clean this up.